spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    # 既にステータスが完了のタスク
+    trait :done_task do
```

Comment:
taskのtraitなのでtaskという情報はつけなくても良いかなと思いました。
```suggestion
    trait :done do
```

Reply:
コミットID[a2b19f2](https://github.com/SakitaDaiki0623/rspec_app_exam/pull/1/commits/a2b19f23495b7347c9d368aaef7e89ccbfdd0b5f)で修正しまいた。